src/test/java/com/google/api/codegen/testdata/php_test_library.baseline

```diff
@@ -794,4 +796,73 @@ class LibraryServiceTest extends PHPUnit_Framework_TestCase
         $this->assertTrue($grpcStub->isExhausted());
     }
 
+    /**
+     * @test
+     */
+    public function getBigBookTest()
+    {
+        list($grpcStub, $client) = $this->createStubAndClient([$this, 'createMockLibraryServiceImpl'], 'createLibraryServiceStubFunction');
+
+        $this->assertTrue($grpcStub->isExhausted());
+
+        // Mock response
+        $name2 = "name2-1052831874";
+        $author = "author-1406328437";
+        $title = "title110371416";
+        $read = true;
+        $expectedResponse = new Book();
+        $expectedResponse->setName($name2);
+        $expectedResponse->setAuthor($author);
+        $expectedResponse->setTitle($title);
+        $expectedResponse->setRead($read);
+        $anyResponse = new Any();
+        $anyResponse->setValue($expectedResponse->serialize());
+        $completeOperation = new Operation();
+        $completeOperation->setName('operations/getBigBookTest')->setDone(true)->setResponse($anyResponse);
+        $grpcStub->addResponse($completeOperation);
+
+        // Mock request
+        $formattedName = LibraryServiceClient::formatBookName("[SHELF_ID]", "[BOOK_ID]");
+
+        $response = $client->getBigBook($formattedName);
+        $response->pollUntilComplete();
+        $actualRequests = $grpcStub->getReceivedCalls();
+        $this->assertSame(1, count($actualRequests));
+        list($actualFuncCall, $actualRequestObject) = $actualRequests[0];
```

Comment:
I think constructing a class would be good for better readability, so you can have something like: `$record->actualFuncCall` And for the string, I agree that it makes more sense to assert as a while instead of calling explode.